examples/messaging/src/App.js

```diff
@@ -10,7 +10,7 @@ import {
   InfiniteScrollPaginator,
   MessageInput,
   MessageInputFlat,
-  MessageList,
+  MessageList as TheMessageList,
```

Comment:
do we need this name reassignment?

Reply:
No, not really, it was a convenience for testing. I reverted the messaging app tweaks.

Reply:
great looks good, will merge